specs/management.yml

```diff
@@ -4479,6 +4479,48 @@ paths:
       name: id
       in: path
       required: true
+  /health-check:
+    get:
+      security: []
+      description: Returns OK or error depending on controller health status
+      tags:
+      - Informational
+      summary: Returns OK or error depending on controller health status
+      operationId: healthCheck
+      responses:
+        "200":
+          description: Base empty response
+          schema:
+            $ref: '#/definitions/empty'
+        "503":
```

Comment:
I would expect a 503 to be returned only if the controller couldn't properly respond for some reason. If it is responding with the data requested, a 503 doesn't make sense.